pkgs/os-specific/linux/kernel/patches.nix

```diff
@@ -68,4 +68,9 @@ rec {
       sha256 = "0bp4jryihg1y2sl8zlj6w7vvnxj0kmb6xdy42hpvdv43kb6ngiaq";
     };
   };
+
+  raspberry_pi_wifi_fix = rec {
+    name = "raspberry-pi-wifi-fix";
+    patch = ./raspberry-pi-wifi-fix.patch;
```

Comment:
Can you pull from here? I don't see the patch in master so I assume it is still in some other branch:

https://raw.githubusercontent.com/archlinuxarm/PKGBUILDs/730522ae76aa57b89fa317c5084613d3d50cf3b8/core/linux-aarch64/0005-mmc-sdhci-iproc-handle-mmc_of_parse-errors-during-pr.patch

It was acked by a kernel maintainer so.